fireatlas/preprocess.py

```diff
@@ -207,7 +207,7 @@ def check_preprocessed_file(
     if freq == "monthly":
         return list(set([(t[0], t[1]) for t in needs_processing]))
     else:
-        return list(set([(t[0], t[1], t[2]) for t in needs_processing]))
+        return list(set([(t[0], t[1], t[2], t[3]) for t in needs_processing]))
```

Comment:
Lines 239 and 247 in FireRunDaskCoordinator are `pfp = preprocessed_filename(t, sat, location=location)`. `preprocessed_filename` then parses the `t` tuple in four places: 

`f"{t[0]}{t[1]:02}{t[2]:02}_{t[3]}{suffix}.txt"`

Previously `check_preprocessed_file` was only returning the first three items in the "NRT" condition, which was causing an indexing issue in `preprocessed_filename`. This error should be resolved now with the full tuple being returned.